controllers/datadogagent/merger/fake/PortManager.go

```diff
@@ -0,0 +1,53 @@
+package fake
```

Comment:
nit, could we rename to port_manager.go ? (i have a PR that suggests changing the other filenamess in merger/fake)

Reply:
and the copyrights :)